src/NzbDrone.Api/Authentication/EnableAuthInNancy.cs

```diff
@@ -39,8 +39,8 @@ public void Register(IPipelines pipelines)
                 pipelines.EnableBasicAuthentication(new BasicAuthenticationConfiguration(_authenticationService, "Sonarr"));                
             }
 
-            pipelines.BeforeRequest.AddItemToEndOfPipeline(RequiresAuthentication);
-            pipelines.AfterRequest.AddItemToEndOfPipeline(RemoveLoginHooksForApiCalls);
+            pipelines.BeforeRequest.AddItemToEndOfPipeline((Func<NancyContext, Response>) RequiresAuthentication);
```

Comment:
I think someone did this before and it broke something in production, @markus101 knows the details.